JASP-Engine/JASP/R/common.R

```diff
@@ -1139,5 +1142,48 @@ saveImage <- function(plotName, format, height, width){
 	
 	updater() # create empty progressbar
 	
+	if (parallel)
+		return(structure(list(updater=updater), class="JASP-progressbar"))
+	
 	return(updater)
 }
+
+.updateParallelProgressbar <- function(progressbar, cluster, results=NULL, complete=FALSE) {
```

Comment:
Can you please document this function? A one line explanation about the different parameters would be awesome.

Reply:
I'll also add some description in common.R in addition to the .md file.